### PR TITLE
DOCSP-36627: Fix double backslashes

### PR DESCRIPTION
**Pull request by norareidy**
JIRA - https://jira.mongodb.org/browse/DOCSP-36627
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-36627-staging/

### CODE REVIEW

**Review by jordan-smith721 — APPROVED** [review]
LGTM!

**Review by GromNaN — APPROVED** [review]
The CI is failing because this is an unmaintained branch.

The doc changes SGTM.

File: docs/reference/method/MongoDBCollection-explain.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@ Definition
 Parameters
 ----------
 
-``$explainable`` : :phpclass:`MongoDB\\Operation\\Explainable`
+``$explainable`` : :phpclass:`MongoDB\Operation\\Explainable`
```
Comment (jmikola) [inline docs/reference/method/MongoDBCollection-explain.txt:32]: @norareidy I think `\\Explainable` was missed here. Just caught it while merging v1.16 to v1.17 and grepping for "\\" in the docs folder.

Comment (jmikola) [reply docs/reference/method/MongoDBCollection-explain.txt:32]: https://github.com/mongodb/mongo-php-library/pull/1244

**Pull request merged**